langkit/lkt_lowering.py

```diff
@@ -248,9 +258,9 @@ class FlagAnnotationSpec(AnnotationSpec):
     """
     Convenience subclass for flags.
     """
-    def __init__(self, name: str):
+    def __init__(self, name: str, default_value: Any = False):
```

Comment:
I think this is hackish. A flag is either passed or not. Here you're trying to conflate a syntactic attribute (presence or not of the flag) into its underlying meaning (whether the property is public or not), except this info is already stored in its semantic state in `PropertyDef`, so no need to do it here. Also you're adding a param to a "flag" spec that doesn't make a lot of sense IMHO...

A simpler and cleaner design IMO is to just pass `annotations.export or None` at the single spot where you need to convert this info for PropertyDef.